akka-projection-jdbc/src/test/scala/akka/projection/jdbc/JdbcProjectionSpec.scala

```diff
@@ -974,7 +996,7 @@ class JdbcProjectionSpec
           .withSaveOffset(1, 1.minute)
 
       projectionTestKit.run(projection) {
-        projectedValueShouldBe("abc|def|ghi|jkl|mno|pqr")
+        projectedValueShouldBe("e1|e2|e3|e4|e5|e6")
```

Comment:
Values in lines 969 and 999 differ on an extra `"|'`. I guess it's correct because the test passes so, unless one of the assertions is secretly ignored or silently failing it'd be good to comment why they are different.

Reply:
This is indeed confusion, but its correct. 

There are two cases in which we are using a async handler. Those handlers append the string on a StringBuffer and it not taking that in consideration. It's always appending a `|`.